docs/gitops-overview.md

```diff
@@ -6,7 +6,7 @@
 
 ## What is GitOps?
 
-GitOps is the practice of automating infrastructure management using Git as a single source of truth.
+GitOps is the practice of automating infrastructure management using Git as a single source of truth.  GitOps was invented in 2017 by [Alexis Richardson, CEO of WeaveWorks](https://www.linkedin.com/in/richardsonalexis/) as a way to do Kubernetes cluster management and application delivery. NubesGen takes this further by replacing the Kubernetes with Terraform, giving you more power to go beyond Kubernetes to everything that Terraform can do.
```

Comment:
I didn't know the term was invented by this person, so thank you for pointing that out.
Then I wouldn't say that NubesGen takes it further, this is quite a common approach to use Terraform to do this.

Reply:
Let's not leave this PR pending: I'm going to merge it and rephrase the second part. Thanks @edburns !